src/components/course-listing.cjsx

```diff
@@ -94,7 +94,7 @@ CourseListing = React.createClass
       </BS.Row>
 
   render: ->
-    courses = CourseListingStore.allCourses() or []
+    courses = CourseListingStore.allCoursesWithRoles() or []
     body = if courses.length
```

Comment:
Pretty sure that `CourseListingStore.allCoursesWithRoles` will always return an array, since it's returning the results of a `_.compact`, which will do that.